app/components/agenda/agenda-header/agenda-version-actions.js

```diff
@@ -86,8 +107,8 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
       isSessionClosable &&
       this.currentSession.isAdmin &&
       this.currentAgendaIsLatest &&
-      this.args.currentAgenda.isDesignAgenda
-    );
+      this.isDesignAgenda
+    )
```

Comment:
bit out of scope, but this getter `canReopenPreviousAgenda` has 1 incorrect check:
`!this.currentSession.isFinal`. This should be `!meeting.isFinal`

Would also prefer `isMeetingClosable`  (session was used for both meeting and current user session in legacy code, snippets of this remain)

If not done here, no worries, will do this on DEV or in small PR afterwards